packages/plugin-audio/web/src/components/SoundControl.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from '../../statics/styles/audio.scss';
+import { SoundIcon, SoundMutedIcon } from 'wix-rich-content-ui-components';
+import AudioSlider from './AudioSlider';
+
+const SoundControl = ({ volume, setVolume, muted, handleMute, handleVolumeChange }) => {
+  const onKeyPress = e => {
+    if (e.key === 'Enter') {
+      handleMute();
+    }
+  };
+
+  const onKeyDown = e => {
+    const shouldDecrease = e.keyCode === 40 && volume > 0;
```

Comment:
```suggestion
    const shouldDecrease = e.key
   Code === 'ArrowDown' && volume > 0;
```

I think this is more easy to understand

Reply:
or you can add it to `KEYS_CHARCODE`
 from 'wix-rich-content-editor-common';